athlinks-ontology.ttl

```diff
@@ -367,10 +367,30 @@ dc:date rdf:type owl:AnnotationProperty .
             rdfs:label "Swim Action"@en .
 
 
-###  http://schema.org/GenderType
-schema:GenderType dc:creator "nextdude"^^xsd:string ;
-                  dc:date "2017-05-22T06:00:00Z"^^xsd:dateTime .
 
+###  http://adp.athlinks.com/schema/BracketRanking
+:BracketRanking rdf:type owl:Class ;
+  rdfs:subClassOf schema:Intangible ;
+  rdfs:comment "An individual result's ranking in a bracket."@en ;
+  rdfs:label "Bracket Ranking" .
+  # Expected properties: 
+  #  schema:disambiguatingDescription -> string (e.g. "M20-29")
+  #  :rank -> Int
+
+:bracketRanking a owl:ObjectProperty ;
+  rdfs:domain :AthleteResult ;
+  rdfs:range :BracketRanking ;
+  rdfs:comment "A bracket ranking of the individual result."@en ;
+  rdfs:label "bracket ranking"@en .
+
+###  http://adp.athlinks.com/schema/rank
+:rank a owl:DatatypeProperty ;
+  rdfs:domain :BracketRanking ;
+  rdfs:range schema:Int ;
+  rdfs:comment "The numeric rank of an individual in a ranking."@en ;
+  rdfs:label "rank"@en .
+
+###  http://adp.athlinks.com/schema/ActivityType
 :ActivityType
```

Comment:
I really want to get rid of `ActivityType`. It was a hack to support the early work in the `adp-api`, but its not necessary. I envision our event types to be subclasses of `ExerciseAction` and we can use the `potentialAction` property on `Thing` to support describing the type of events, races, and results.